posts/2021-03-13-microprofile-config-2.0.adoc

```diff
@@ -104,36 +105,47 @@ The examples below use the config property values defined in this example META-I
 .META-INF/microprofile-config.properties
 [source]
 ----
-%testing.server.host= example.test.org
+%testing.server.host=example.test.org
+%production.server.host=example.prod.org
 server.host=example.org
 server.port=9080
 server.url=http://${server.host}
 ----
 
 [#Config-Profile]
 === Config Profile
-link:https://download.eclipse.org/microprofile/microprofile-config-2.0/microprofile-config-spec-2.0.html#configprofile[Config Profiles] allow configuration for different environments and development stages while only one of them is active (e.g. dev, test, prod). The Config Profile is specified using the `mp.config.profile` property, which can be set in any ConfigSource. Once it is set, the corresponding properties associated with the active profile are used. The value can also be set at application start-up, for example if you’re using Maven and the link:https://github.com/OpenLiberty/ci.maven[liberty-maven-plugin] you can start your app in link:https://openliberty.io/docs/21.0.0.2/development-mode.html[Dev mode] with:
+link:https://download.eclipse.org/microprofile/microprofile-config-2.0/microprofile-config-spec-2.0.html#configprofile[Config Profiles] allow configuration for different environments and development stages while only one of them is active (e.g. dev, test, prod). The Config Profile is specified using the `mp.config.profile` property, which can be set in any ConfigSource. Once it is set, the corresponding config property values associated with the active profile are used. The `mp.config.profile` property can also be set at application start-up, for example if you’re using Maven and the link:https://github.com/OpenLiberty/ci.maven[liberty-maven-plugin] you can start your app in link:https://openliberty.io/docs/latest/development-mode.html[Dev mode] with:
```

Comment:
```suggestion
link:https://download.eclipse.org/microprofile/microprofile-config-2.0/microprofile-config-spec-2.0.html#configprofile[Config Profiles] allow configuration for different environments and development stages while only one of them is active (e.g. dev, test, prod). The active Config Profile is specified using the `mp.config.profile` property, which can be set in any ConfigSource. Once it is set, the corresponding config property values associated with the active profile are used. The `mp.config.profile` property can also be set at application start-up, for example if you’re using Maven and the link:https://github.com/OpenLiberty/ci.maven[liberty-maven-plugin] you can start your app in link:https://openliberty.io/docs/latest/development-mode.html[Dev mode] with:
```

Reply:
Not sure if anything needs to be changed but... Setting the property at startup isn't really an alternative method since you're really just setting a value into a builtin ConfigSource

Reply:
Yeah I agree. I think it's trying to say: "here's an example of where you might want to set the property", rather than "alternatively to setting it in your application, you can set it here". I can update this if it isn't clear?